woocommerce/woocommerce
# [e2e tests] Fix blocks tests failing with Gutenberg active

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).

### Changes proposed in this Pull Request:

Fixes blocks e2e tests that are failing when Gutenberg is active and the editor canvas is within an iframe.


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the e2e tests that are using the editor are passing with and without Gutenberg. 
2. Check CI, all tests should pass. See test runs: [with Gutenberg](https://github.com/woocommerce/woocommerce/actions/runs/8805465182/job/24168838038?pr=46842), without Gutenberg.
